### ISSUE TITLE
Tasks do not appear after sucessful import

### ISSUE THREAD

**Issue by lefterav** [choko/MT-ComparEval#31]
I just installed the latest version of ComparEval. I created a folder structure similar to the one I had previously and I run watcher. The watcher run without a problem and it gave the messages that all experiments and tasks have been imported. But the graphical interface only lists the experiments, but when I click to every experiments, no tasks appear.

**Comment by choko** [comment]
What is the output of `find data -name ".imported"`?
What is the output of `sqlite3 ./storage/database "SELECT name FROM tasks"`?

**Comment by choko** [comment]
Also, can you check, that there are no errors in Google Chrome Developer Console?

**Comment by lefterav** [comment]
```
$ find data -name ".imported"
data/en-de-batch1a/pilot0_2015-07-28_171832/.imported
data/en-de-batch1a/.imported
data/en-de-batch1a/pilot1_2015-07-28_171832/.imported
data/en-de-batch2a/pilot0_2015-07-28_171832/.imported
data/en-de-batch2a/.imported
data/en-de-batch2a/pilot1_2015-07-28_171832/.imported
data/de-en-batch1q/.imported
data/de-en-batch2q/pilot0_2015-07-28_171832/.imported
data/de-en-batch2q/.imported
data/de-en-batch2q/pilot1_2015-07-28_171832/.imported
```

```
$ sqlite3 ./storage/database "SELECT name FROM tasks"
Pilot 0.00
Pilot 1.00
Pilot 0.00
Pilot 1.00
Pilot 0.00
Pilot 1.01
```

**Comment by lefterav** [comment]
Oh yes, the javascript console gives this error:
```
ross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://lnv-3209.sb.dfki.de:8070/evaluation/api/metrics/scores-in-experiment?experimentId=3. (Reason: CORS header 'Access-Control-Allow-Origin' missing).
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://lnv-3209.sb.dfki.de:8070/evaluation/api/metrics/scores-in-experiment?experimentId=3. (Reason: CORS request failed).
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://lnv-3209.sb.dfki.de:8070/evaluation/api/tasks?experimentId=3. (Reason: CORS header 'Access-Control-Allow-Origin' missing).
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://lnv-3209.sb.dfki.de:8070/evaluation/api/tasks?experimentId=3. (Reason: CORS request failed).
```

the problem is that the host name of the server I am using is `lnv-3209.sb.dfki.de:8070` but the external users can only access this through a proxy workbench.qtleap.eu. This is a necessary configuration for our network. I think I had figured out how to fix this.

**Comment by lefterav** [comment]
the problem occurs because the $baseUri variable is not defined properly. A possible solution would be to not get the $baseUri from the server, but instead call the js files through a relative path (e.g. '../js/directives.js'

I had fixed it in a similar manner, in the previous version. Unfortunately I have to leave the office now. I can work on a solution tomorrow, unless you would like to fix it now.

**Comment by choko** [comment]
This was fixed in #6 but it is not merged into master yet. Can you check branch https://github.com/choko/MT-ComparEval/tree/base_path?

**Comment by lefterav** [comment]
Closed as duplicate of #6
